File: docs/components_page/components/button/usage.R
library(dashBootstrapComponents)

button <- div(
  list(
    dbcButton("Click me",
      id = "example-button", n_clicks = 0,
      className = "me-2"
    ),
    span(id = "example-output", style = list(verticalAlign = "middle"))
  )
)


app$callback(
  output("example-output", "children"),
  list(input("example-button", "n_clicks")),
  function(n) {
    if (n > 0) {
      return(sprintf("Clicked %d times.", n))
    }
    return(sprintf("Not clicked"))
  }
)

File: docs/components_page/components/button/usage.jl
using DashBootstrapComponents

button = html_div([
    dbc_button("Click me", id = "example-button", className = "me-2", n_clicks = 0),
    html_span(id = "example-output", style = Dict("verticalAlign" => "middle")),
]);

callback!(
    app,
    Output("example-output", "children"),
    Input("example-button", "n_clicks"),
) do n
    return n > 0 ? "Clicked $n times." : "Not clicked."
end;
